src/CADL.Extension/Emitter.Csharp/src/type/BodyMediaType.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export enum BodyMediaType {
```

Comment:
I think we need to align to https://swagger.io/docs/specification/2-0/mime-types/

Reply:
I will discuss with Alexandar about it.  And may update late.